test/e2e/main_test.go

```diff
@@ -27,13 +28,19 @@ import (
 
 // channelTestMap indicates which test cases we want to run for a given CCP.
 var channelTestMap = map[string][]func(t *testing.T){
-	"in-memory-channel": []func(t *testing.T){
+	test.InMemoryChannelProvisioner: []func(t *testing.T){
```

Comment:
Sorry, I missed this in the last review. We need to be testing with `in-memory`. We can also test `in-memory-channel`, but `in-memory` is the one the recommended one.

Reply:
Done.